pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.609.1</version>
```

Comment:
[CloudBees hat]
1.565.1 was 2014/07/30 which is less than 7 days away... all hail 1.580 as the last line of supported LTS lines
[/CloudBees hat]

I'm fine with that technical reason for bumping to first LTS > 1.568